redbox/llm/llm_base.py

```diff
@@ -147,7 +145,55 @@ def chat_with_rag(
             },
             callbacks=callbacks or [],
         )
-        return result, docs_with_sources_chain
+        return result
+
+    def stuff_doc_summary(
+        self, prompt: PromptTemplate, documents: list[Document], user_info: dict, callbacks: Optional[list] = []
```

Comment:
```suggestion
        self, prompt: PromptTemplate, documents: list[Document], user_info: dict, callbacks: Optional[list] = None
```
same reason as above